docs-ref-mapping/reference-unified.yml

```diff
@@ -1424,5 +1429,29 @@
     - name: Other
       uid: azure.nodejs.sdk.landingpage.services.other
       landingPageType: Service
-      children:
-      - '**'
+      items:
+      - name: Abort Controller
+        uid: azure.nodejs.sdk.landingPage.services.other.abortcontroller
+        href: ~/docs-ref-services/{moniker}/abort-controller-readme.md
+        children:
+        - '@azure/abort-controller'
+      - name: Core AMQP
+        uid: azure.nodejs.sdk.landingPage.services.other.coreamqp
+        href: ~/docs-ref-services/{moniker}/core-amqp-readme.md
+        children:
+        - '@azure/core-amqp'
+      - name: Core Auth
+        uid: azure.nodejs.sdk.landingPage.services.other.coreauth
+        href: ~/docs-ref-services/{moniker}/core-auth-readme.md
+        children:
+        - '@azure/core-auth'
+       - name: Core Auth
```

Comment:
Isn't this a duplicate entry of above core auth?

Reply:
This looks incomplete. Something is wrong with the commit history